spec/spec_helper.rb

```diff
@@ -1,10 +1,10 @@
 require 'simplecov'
 require 'simplecov-console'
 require 'capybara/rspec'
-
+require_relative './setup_test_database'
 
 # Set the environment to "test"
-ENV['RACK_ENV'] = 'test'
+ENV['ENVIRONMENT'] = 'test'
```

Comment:
I also used ENVIRONMENT, would be curious to know the difference between this and RACK_ENV